packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -473,6 +472,41 @@ export class Users {
       }
     );
 
+    // Set unclaimedProceeds & unclaimedProfit
+    const marketsData = await db.findMarkets({
```

Comment:
really we should just replace the findMarketCreatedLogs way above with this

Reply:
Yeah it's not needed anymore. I'll remove it.